src/comp/comp.py

```diff
@@ -24,48 +24,51 @@ def __repr__(self):
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with 'D':
 print("Starts with D:")
-a = []
+a = [x.name for x in humans if x.name.startswith("D")]
 print(a)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name ends in "e".
 print("Ends with e:")
-b = []
+b = [x.name for x in humans if x.name.endswith("e")]
 print(b)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with any letter between 'C' and 'G' inclusive.
 print("Starts between C and G, inclusive:")
-c = []
+# create letters list to check against
+letters =["C", "D", "E", "F", "G"]
+# use startswith method and tuple of letters
+c = [x.name for x in humans if x.name.startswith(tuple(letters))]
```

Comment:
nice. there´s also an ordinal relation between letters in Python, so
'A' < 'B' (is true)
'X' > 'Y' (returns false)
so the code below also works

c = [h.name for h in humans if h.name[0] >= 'C' and h.name[0] <= 'G']